src/components/Navbar.tsx

```diff
@@ -28,8 +28,12 @@ export default function Navbar() {
     </nav>
   );
 }
+interface BurgerMenuButtonProps {
+  open: boolean;
+  onToggle: () => void;
+}
 
-function BurgerMenuButton({ open, onToggle }) {
+function BurgerMenuButton({ open, onToggle }: BurgerMenuButtonProps) {
```

Comment:
It might be worth considering taking parameters out of the object. While on the one hand we achieve encapsulation of parameters, on the other we have to create a separate object for each component, which is tedious if you just need children prop for example. 
`function BurgerMenuButton(open: boolean, onToggle: [whatever type functions are]) {` //alternative
Not saying either approach is better